test/e2e/case13_templatization_test.go

```diff
@@ -105,6 +105,8 @@ var _ = Describe("Test templatization", func() {
 
 				return utils.GetComplianceState(managedPlc)
 			}, defaultTimeoutSeconds, 1).Should(Equal("Compliant"))
+
+			By("Clean up")
```

Comment:
Should this be in an `AfterAll()` function?

Reply:
Also last spec